examples/ecs/system_closures.rs

```diff
@@ -0,0 +1,28 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_system(Box::new(move |mut cmd: Commands, arg: Local<String>| {
```

Comment:
This is more of a question, but is the `move` necessary here?

Reply:
good catch, it's no longer necessary. will remove